### DISCUSSION TITLE
ScriptReference/EditorUtility.SaveFilePanel

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#158] [Notes]
# ScriptReference/EditorUtility.SaveFilePanel

https://docs.unity3d.com/ScriptReference/EditorUtility.SaveFilePanel.html

**Comment by vertxxyz** [comment]
This returns a full path. If you want a path local to the project, use [`EditorUtility.SaveFilePanelInProject`](https://docs.unity3d.com/ScriptReference/EditorUtility.SaveFilePanelInProject.html) instead.
Be wary, it has different arguments.
